packages/metal-quartz-icon/src/MetalQuartzIcon.soy

```diff
@@ -1,17 +1,57 @@
 {namespace MetalQuartzIcon}
 
 /**
- * This renders an icon element based on the lexicon icon definition
- * found at http://liferay.github.io/lexicon/content/icons-lexicon/
+ * This renders an icon element based on the Quartz Icon, Quartz Font Awesome
+ * Icon, and Glyphicon definition.
+ * http://getbootstrap.com/components/#glyphicons
  *
  * @param? elementClasses
  * @param spritemap
  * @param symbol
  */
 {template .render}
 	{if isNonnull($symbol) and isNonnull($spritemap)}
-		<svg aria-hidden="true" class="lexicon-icon lexicon-icon-{$symbol}{$elementClasses ? ' ' + $elementClasses : ''}">
-			<use xlink:href="{$spritemap}#{$symbol}" />
-		</svg>
+		{let $quartzIconElementClasses kind="text"}
+			{$elementClasses ? ' ' + $elementClasses : ''}
+		{/let}
+
+		{let $quartzIconVariant kind="text"}
+			{$spritemap == 'quartz-fa' ? 'quartzFa' : $spritemap}
+		{/let}
```

Comment:
Hey @jbalsas 

You actually can't reassign params in soy, and you can't reassign `let` values either.

+1 for `@param iconLibrary`.

I also don't think `fa` is too big of a problem since font awesome itself uses it all the time.

`<i class="fa fa-address-book" aria-hidden="true"></i>`